CONTRIBUTING.md

```diff
@@ -134,17 +134,20 @@ onwards and Eclipse 2020-3 and onwards.
     ./gradlew :run
 
 That'll spend a while building Elasticsearch and then it'll start Elasticsearch,
-writing its log above Gradle's status message. We log a lot of stuff on startup,
-specifically these lines tell you that Elasticsearch is ready:
+writing its log above Gradle's status message. We log a lot of stuff on startup, when you see some lines like these ones (when the console stops scrolling):
 
-    [2020-05-29T14:50:35,167][INFO ][o.e.h.AbstractHttpServerTransport] [runTask-0] publish_address {127.0.0.1:9200}, bound_addresses {[::1]:9200}, {127.0.0.1:9200}
-    [2020-05-29T14:50:35,169][INFO ][o.e.n.Node               ] [runTask-0] started
```

Comment:
I see that we end with the security line now. It is a bit arbitrary and somewhat likely to change. Therefore I would like to also keep this line. I propose to keep both lines, with `...` between them to signal that other stuff goes in between.